### PR TITLE
DISCO-656: Adds in search to dashboard

### PR DESCRIPTION
**Pull request by jlajoie**
Adds in search bar to dashboard, depends on course-discovery API accepting `?pubq=`

Without search applied:
Screen Shot 2019-07-09 at 1 12 43 PM

With search applied:
Screen Shot 2019-07-09 at 1 12 50 PM

### CODE REVIEW

File: src/data/actions/table.js [hunk 1]
```diff
@@ -88,50 +116,14 @@ const paginateTable = (tableId, fetchMethod, pageNumber) => (
   }
 );
 
-const sortBy = (dataToSort, orderField) => {
```
Comment (jlajoie) [inline src/data/actions/table.js]: Wasn't used because we never set num_pages

File: src/data/actions/table.js [hunk 2]
```diff
@@ -88,50 +116,14 @@ const paginateTable = (tableId, fetchMethod, pageNumber) => (
...
-    const tableState = getState().table[tableId];
+  (dispatch) => {
     const options = {
       ...getPageOptionsFromUrl(),
       ordering,
     };
     dispatch(sortRequest(tableId, ordering));
 
-    // If we can sort client-side because we have all of the data, do that
-    if (tableState.data && tableState.data.num_pages === 1) {
```
Comment (jlajoie) [inline src/data/actions/table.js]: Wasn't used because we never set num_pages

File: src/components/ButtonToolbar/index.jsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import classNames from 'classnames';
 const ButtonToolbar = (props) => {
   const { className } = props;
   const fullClassName = classNames(
-    'btn-toolbar justify-content-end',
+    'btn-toolbar',
```
Comment (mikix) [inline src/components/ButtonToolbar/index.jsx]: Dropping that might change all the other toolbars on the site. We use it on the course edit page and the course run collapsibles, I think.

Comment (jlajoie) [reply src/components/ButtonToolbar/index.jsx]: Oh good catch, I should have seen if this had cascading changes, I'll look for a better way to override

**Pull request merged**